src/tooltip.jsx

```diff
@@ -23,39 +23,61 @@ class Tooltip extends Component {
     shouldShow: false,
   }
 
-  render () {
-    const child = Children.only(this.props.children)
+  child = Children.only(this.props.children)
+
+  constructor (props) {
+    super(props)
+    this._onMouseOver = this._onMouseOver.bind(this)
+    this._onMouseOut = this._onMouseOut.bind(this)
+  }
 
+  render () {
     return (
       <span className={this.props.wrapperClassName}>
-        {cloneElement(child, {
+        {cloneElement(this.child, {
           ref: (node) => {
             this._target = node
-            if (typeof child.ref === 'function') {
-              child.ref(node)
+            if (typeof this.child.ref === 'function') {
+              this.child.ref(node)
             }
           },
-          onMouseOver: (...args) => {
-            child.props.onMouseOver && child.props.onMouseOver(...args)
-            this.setState({ shouldShow: true })
-          },
-          onMouseOut: (...args) => {
-            child.props.onMouseOut && child.props.onMouseOut(...args)
-            this.setState({ shouldShow: false })
-          },
+          onMouseOver: this._onMouseOver,
+          onMouseOut: this._onMouseOut,
         })}
         {this._popper()}
       </span>
     )
   }
 
+  _onMouseOver (e, ...args) {
+    this.child.props.onMouseOver && this.child.props.onMouseOver(e, ...args)
+    this.setState({ shouldShow: true })
+  }
+
+  _onMouseOut (e, ...args) {
+    if (this.props.clickable) {
```

Comment:
We might want to just always keep the tooltip around if you hover over it, so we wouldn't even need the `clickable` option.

Reply:
Good point, changed